ansible/roles/docker/templates/etc/default/docker.default

```diff
@@ -4,8 +4,11 @@
 #DOCKER="/usr/local/bin/docker"
 
 # Use DOCKER_OPTS to modify the daemon startup options.
-#DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
-DOCKER_OPTS="--storage-driver=devicemapper"
```

Comment:
We run around 10-15 - I think we run anywhere from 450-500 docker containers in a cluster of 35 minions (c4.2xlarge). We'd probably run more if our main API/mega-ruby-app didn't need 6 gigs of RAM to run :)